src/styles/blocks/card.scss

```diff
@@ -0,0 +1,105 @@
+.card {
+  display: flex;
+  flex-direction: column;
+
+  box-sizing: border-box;
+
+  width: 200px;
+  padding: 32px 0 16px;
+  margin-right: 48px;
+  margin-bottom: 46px;
+
+  border: 1px solid $color-close-to-white;
+  border-radius: 5px;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &:nth-last-child(-n+4) {
+    margin-bottom: 0;
+  }
+
+  &__image {
+    @include size-y-x(134px, 160px);
+
+    padding: 0 19px;
+    margin-bottom: 40px;
+  }
+
+  &__info {
+    margin-bottom: 4px;
+    padding: 0 16px;
```

Comment:
I think it would be easier to remove these paddings and add horizontal padding to the **.card** and change your padding size of **card__image**

Reply:
It's normal idea but not for this case, cause image has another padding, I have better idea